platform-dom/src/main/java/com/softicar/platform/dom/engine/IDomEngine.java

```diff
@@ -169,9 +171,27 @@ default void setCssClassOnKeyDown(IDomNode node, DomEventType eventType, Collect
 
 	// -------------------------------- pop-ups -------------------------------- //
 
-	void showPopup(IDomNode popup, int x, int y, DomPopupXAlign xAlign, DomPopupYAlign yAlign);
+	/**
+	 * Moves the {@link DomPopup} with the given {@link IDomPopupFrame},
+	 * according to the given parameters.
+	 * <p>
+	 * The {@link DomPopup} shall be appended when this method is called.
+	 * Otherwise, its position might be unexpected.
```

Comment:
why `DomPopup` and what does `appended` mean?
```suggestion
	 * The {@link IDomPopupFrame} must be appended to the {@link DomBody} or one of
	 * its children before this method is called. Otherwise, its position might be unexpected.
```